mmdet/datasets/transforms/wrappers.py

```diff
@@ -48,3 +50,37 @@ def __repr__(self) -> str:
         repr_str = self.__class__.__name__
         repr_str += f'(branch_pipelines={list(self.branch_pipelines.keys())})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class ShuffledSequence(Compose):
```

Comment:
Can support passing in the `num_selected` parameter so that the user can randomly select part of the pipeline and then shuffled

Reply:
In fact, `RandAugment` has already supported randomly selecting part of the pipeline, so `ShuffledSequence` only simplifies the logic.

Reply:
RandomOrder might be a more appropriate name